src/js/views/labels_from_maps_modal.js

```diff
@@ -54,33 +54,46 @@ var LabelFromMapsModal = Backbone.View.extend({
         event.preventDefault();
         if (this.isLoading) return;
 
-        var key = $('select', this.el).val();
+        var radios = $("input[name='kvpChoice']")
+        var choice
+        for (var i = 0, length = radios.length; i < length; i++) {
+            if (radios[i].checked) {
+                choice = radios[i].value;
+                break;
+            }
+        }
+        console.log(choice)
```

Comment:
I was just about to merge this but noticed this `console.log` you might want to remove.
I think you could also just get the choice in a one-liner:
```
var choice = $("input[name='kvpChoice']:checked").val();
```